app/scripts/proactive/view/CatalogGetView.js

```diff
@@ -359,10 +372,13 @@ define(
                 }, this);
                 //
                 if(that.$('#catalog-get-buckets-table tr').length) {
-                    // Select the previous bucket if it isn't the first time, otherwise, select the first bucket on the list
-                    if(localStorage.selectBucket){
+                    // Select the previous bucket from the local storage if it isn't the first time
+                    // else, select the bucket given as input to the render function
```

Comment:
It should be :
Select the bucket given as an input to the render function
else, Select the previous bucket from the local storage if it isn't the first time
otherwise, select the first bucket on the list

Reply:
ok